src/Sparrow/Json/BlittableJsonTextWriter.cs

```diff
@@ -755,6 +755,17 @@ public virtual void Dispose()
             {
                 //we are disposing, so this exception doesn't matter
             }
+            // TODO: remove when we update to .net core 3
+            // https://github.com/dotnet/corefx/issues/36141
+            catch (NotSupportedException e)
+            {
+                if (e.Message.Contains("The WriteAsync method cannot be called when another write operation is pending"))
```

Comment:
Not a good idea, may get different values under different cultures. 
Just wrap the exception

Reply:
Okay, but what if we getting a different `NotSupportedException`? what about to check only for the word `WriteAsync` ?

Reply:
I don't consider this likely, and the idea is that we'll restart anyway

Reply:
okay, done.